src/WebJobs.Script/Utility.cs

```diff
@@ -989,17 +989,22 @@ public static bool IsMediaTypeOctetOrMultipart(MediaTypeHeaderValue mediaType)
             }
         }
 
-        // EnableWorkerIndexing set through AzureWebjobsFeatuerFlag always take precdence
-        // if AzureWebjobsFeatuerFlag is not set then WORKER_INDEXING_ENABLED hosting config controls stamplevel enablement
-        // if WORKER_INDEXING_ENABLED is set and WORKER_INDEXING_DISABLED contains the customers app name worker indexing is then disabled for that customer only
-        // Also Worker indexing is disabled for Logic apps
+        // Worker indexing is disabled for Logic apps
```

Comment:
Can you also add in comment here that WorkerIndexing is enabled by default (unless explicitly disabled by hosting config - this is already mentioned in the comment above) and about the precedence taken by WorkerConfig as demonstrated in the tests? So that we have all these conditions explained at one place.

Reply:
That is a good point to make note of. I will add it.